setup.py

```diff
@@ -32,5 +33,8 @@
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python:: Implementation:: CPython',
```

Comment:
missing space before `::` (also in the next line)

Reply:
Thanks, reset, fixed, and force pushed.